src/spawn/arguments.ts

```diff
@@ -0,0 +1,64 @@
+const _ = require('underscore');
+const checkTypes = require('check-types');
+
+import { CanDeployOptions } from '../can-deploy';
+import { MessageOptions } from '../message';
+import { PublisherOptions } from '../publisher';
+import { ServiceOptions } from '../service';
+import { VerifierOptions } from '../verifier';
+
+export type SpawnArguments =
```

Comment:
What's the idea behind it being allowed to be an empty object?

Reply:
It made the tests significantly less noisy.

I thought it was safe to extend the type this way, because it's not exposed. But I haven't written much TypeScript, so it might be bad practice.

For now I'll put a comment in the definition.